5eShaped/translations/fr.json

```diff
@@ -543,6 +546,8 @@
   "REACTION":"Réaction",
   "REACTIONS":"Réactions",
   "RECHARGE":"Recharge",
+  "REDUCED_MAX":"Maximum réduit",
```

Comment:
I like "Max réduit" here; it is obvious that it concerns HP, for the heart shape of the button.